### ISSUE TITLE
Comparison with chrischristo/capsule-maven-plugin 

### ISSUE THREAD

**Issue by thiloplanz** [pguedes/capsule-maven-plugin#3]
How does this plugin compare to the other one found at https://github.com/chrischristo/capsule-maven-plugin ?

Are they related? Which one should I use?

One thing I noticed is that the @chrischristo one requires Maven 3.1 (https://github.com/chrischristo/capsule-maven-plugin/issues/4), while the @pguedes one seems to work with 3.0, too.

**Comment by thiloplanz** [comment]
> One thing I noticed is that the @chrischristo one requires Maven 3.1 while the @pguedes one seems to work with 3.0, too.

That is for version 0.1 only. 
From 0.2, the same error (chrischristo/capsule-maven-plugin#4) occurs here, too, if you try to run it with Maven 3.0.

**Comment by chrischristo** [comment]
Hi @thiloplanz, both plugins aim to do the same thing. However the [chrischristo plugin]( https://github.com/chrischristo/capsule-maven-plugin) which I wrote with help from others has been in more active development and supports all the latest features of capsule. I will continue to update the plugin to support any new features [Capsule](https://github.com/puniverse/capsule) comes up with. And yes the chrischristo plugin requires maven 3.1+ because it uses the new Eclipse Aether plugin to do its magic ([see a previous issue on this here](https://github.com/chrischristo/capsule-maven-plugin/issues/4)).

**Comment by pguedes** [comment]
Hi,

@chrischristo <https://github.com/chrischristo>'s version has support
for more features... mine is simpler.

I started mine because there wasn't any and i needed one... then he did
his... then i looked at his and thought i did not need the extra
features and found my code easier to maintain... so I kept mine but only
with the features i need.

I use it for a work project so it will grow as i need it but im not very
interested in developing features i do not need as i would have to
maintain those also...

So mine is simpler... his has more features.

hope this helped,
Pedro

**Comment by pguedes** [comment]
yep... same here... 3.1 requirement is to use Aether to get latest
capsule from a maven repo...
